asv_bench/benchmarks/benchmarks.py

```diff
@@ -1184,4 +1184,96 @@ def time_series_indices(self, shape):
         self.series.groupby(self.series).indices
 
 
+class TimeRepr:
+
+    params = [get_benchmark_shapes("TimeRepr")]
+    param_names = ["shape"]
+
+    def setup(self, shape):
+        self.df = IMPL.DataFrame(np.random.randn(*shape))
+        execute(self.df)
+
+    # returns a  string thus not calling execute
+    def time_repr(self, shape):
+        repr(self.df)
+
+
+class TimeMaskBool:
+
+    params = [get_benchmark_shapes("TimeMaskBool")]
+    param_names = ["shape"]
+
+    def setup(self, shape):
+        df = IMPL.DataFrame(np.random.randn(*shape))
+        df = df.where(df > 0)
+        self.bools = df > 0
+        self.mask = IMPL.isnull(df)
+        execute(self.bools), execute(self.mask)
+
+    def time_frame_mask_bools(self, shape):
+        execute(self.bools.mask(self.mask))
+
+    def time_frame_mask_floats(self, shape):
+        execute(self.bools.astype(float).mask(self.mask))
+
+
+class TimeIsnull:
+
+    params = [get_benchmark_shapes("TimeIsnull")]
+    param_names = ["shape"]
+
+    def setup(self, shape):
+        self.df_no_null = IMPL.DataFrame(np.random.randn(*shape))
+        sample = np.array([np.nan, 1.0])
+        data = np.random.choice(sample, (shape[0], shape[1]))
+        self.df = IMPL.DataFrame(data)
+        execute(self.df), execute(self.df_no_null)
+
+    def time_isnull_floats_no_null(self, shape):
+        execute(IMPL.isnull(self.df_no_null))
```

Comment:
Let's drop "no_null" case. No big difference for Modin.

```suggestion
        sample = np.array([np.nan, 1.0])
        data = np.random.choice(sample, (shape[0], shape[1]))
        self.df = IMPL.DataFrame(data)
        execute(self.df)
```